tests/InputFiles/ScalarWave/Input2DPeriodic.yaml

```diff
@@ -37,11 +37,10 @@ DomainCreator:
     InitialRefinement: [2, 2]
     InitialGridPoints: [4, 4]
 
-# If filtering is enabled in the executable the filter can be controlled using:
-# Filtering:
-#   ExpFilter0:
-#     Alpha: 12
-#     HalfPower: 32
+Filtering:
```

Comment:
Please add the same warning comment as in the executable source file.

(To be clear: my worry is that someone could run 2D numerical experiments and not realize there is an unneeded filter present in the code)